modules/ROOT/pages/edit-exchange-json-file.adoc

```diff
@@ -0,0 +1,30 @@
+= Edit Your API Specification Project Exchange File
+
+Anypoint Studio enables you to edit the `exchange.json` file of your API specification project. The `exchange.json` file is a file that contains metadata required by Anypoint Exchange to describe your API asset.
```

Comment:
The `exchange.json` file is a file that contains metadata (sounds redundant) > The `exchange.json` file contains **the** metadata...